### ISSUE TITLE
Leftover HTML tags when clearing out editor content

### ISSUE THREAD

**Issue by fahmilatib** [Alex-D/Trumbowyg#257]
### Informations

**Note: This issue only happens when working in visual mode, not HTML mode**

I have a `<textarea>` with pre-populated content like this (simplified):

```
<textarea>
    &lt;p&gt;Vel quis ratione hic ratione quasi mollitia. Soluta modi qui ut alias accusantium. Sed quod voluptatem incidunt dolor itaque. Quis aut quae iusto adipisci id.&lt;/p&gt;
</textarea>
```

* `&lt;p&gt;` equals `<p>`
* `&lt;/p&gt;` equals `</p>`

If I'm on **visual mode** and I clear out the content either by <kbd>CTRL</kbd>+<kbd>A</kbd> or manually highlighting all, and then delete, it leaves
```
<p><br></p>
```
in the **HTML mode** (you can't see those tags in visual mode obviously).

Well as you can see empty tags don't make sense really. This will cause my php form validation to treat the field as **not empty**. Of course I can always check for those leftover tags but I don't think that will be an elegant solution since they should be trimmed out by Trumbowyg in the first place.

Note that as I've mentioned earlier, this issue won't happen if I clear out the content using HTML mode.

**Browser + version**: Chrome 50.0.2661.75 m
**OS**: win 10
**Resolution**: any

### How to reproduce the bug?

Go to Trumbowyg [homepage](http://alex-d.github.io/Trumbowyg/) and do the exact same steps that I've described above. However, your result will be **slightly different** due to the difference in the pre-populated content.

So in short, you will likely get
```
<h3><br></h3>
```

instead of
```
<p><br></p>
```

**Comment by AdamHess** [comment]
It sounds like you're looking for an option like. "trimNonvisibleHtml" 

a quick solution for you right now would be something like this. 
```
//get the text
var text = $('textarea.mytext').text() 
//this will replace all HTML tags with empty space
//trim for good measure
var cleanedText = text.replace(/<[^>]*>/g, "").trim();
if(cleanedText) {
    console.log("text is not empty"); 
}
```

**Comment by fahmilatib** [comment]
Pretty neat solution over there :)

Still hoping for a built-in function though, but until then yours would be perfect. Thanks.

**Comment by Alex-D** [comment]
It seems to be OK since 2.1.0 :)

**Comment by fahmilatib** [comment]
Thank you!
